_meta/config/common.p2.yml.tmpl

```diff
@@ -66,6 +66,15 @@ inputs:
 #   # The name of the output to use for monitoring data.
 #   use_output: monitoring
 #   # exposes agent metrics using http, by default sockets and named pipes are used
+#   #
+#   # `http` Also exposes a /liveness endpoint that will return an HTTP code depending on agent status:
+#   # 200: Agent is healthy
+#   # 500: A component or unit is in a failed state
+#   # 503: The agent coordinator is unresponsive
+#   # You can pass a `failon` parameter to the /liveness endpoint to determine what component state will result in a 500.
+#   # For example: `curl 'localhost:6792/liveness?failon=degraded'` will return 500 if a component is in a degraded state.
+#   # The two possible values for `failon` are `degraded` and `failed`. If no `failon` parameter is provided, the default 
```

Comment:
As mentioned later the default should just be the coordinator liveness check, with degraded and failed as optional additional extensions to it.